totaladapters/src/test/java/pl/arturborowy/util/ViewInflaterTest.kt

```diff
@@ -0,0 +1,92 @@
+package pl.arturborowy.util
+
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import android.widget.ImageView
+import android.widget.LinearLayout
+import android.widget.TextView
+import com.nhaarman.mockito_kotlin.given
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.spy
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import org.mockito.MockitoAnnotations
+import org.robolectric.RobolectricTestRunner
+import pl.arturborowy.R
+
+@RunWith(RobolectricTestRunner::class)
+internal class ViewInflaterTest {
+
+    private val viewInflater = ViewInflater()
+
+    private val context = TestData.getContext()
+
+    @Before
+    fun setUp() = MockitoAnnotations.initMocks(this)
+
+    @Test
+    fun `inflateToType with same ViewT as givenLayout's top view type calls inflate with same arguments`() {
+        val givenLayoutResId = R.layout.view
```

Comment:
I accepted it earlier on, but I think this resource shouldn't be in production package. Try to move it to tests.